docs/_plugins/empty-p.cjs

```diff
@@ -0,0 +1,20 @@
+const cheerio = require('cheerio');
+
+/**
+ * @param {import('@11ty/eleventy/src/UserConfig')} eleventyConfig
+ */
+module.exports = function(eleventyConfig) {
+  eleventyConfig.addTransform('delete-empty-p', async function(rawContent, outputPath) {
+    if (!outputPath || !outputPath.endsWith('.html')) {
+      return rawContent;
+    } else {
+      const $ = cheerio.load(rawContent);
+      $('p:empty').each(function() {
+        if (!this.attributes.length) {
```

Comment:
@bennypowers not seeing the issue anymore (lgtm!). For my own learning though, why do we want to keep empty paragraph elements that have attributes?

Reply:
Could be they were put there for a reason? if it has an attr, it's not an artifact of the markdownit ast, so we shouldn't remove it